### PR TITLE
Version bump to 1.5.0

### PR DESCRIPTION
**Pull request by wenningerk**
Bump Version to 1.5.0
    
    Changes since 1.4.2
    
    - default to resource-syncing with pacemaker in spec-file and configure.ac
      This default has to match between sbd and pacemaker and
      thus qualifies this release for a minor-version-bump
    - fix some regressions introduced by adding configurability previously
    - adapt description of startup/shutdown sync with pacemaker
    - make watchdog warning messages more understandable

### CODE REVIEW

**Comment by wenningerk** [comment]
@kgaillot 

Any thoughts about the versioning? Basic approach, clearness in pointing out the impact, ...

**Review by kgaillot — COMMENTED** [review]
Looks good to me. The version bump makes sense.

If you wanted to be safer, you could offer configure/RPM "with" options for the yes/no value, and/or auto-detect the default based on rpm distro macros.

**Comment by wenningerk** [comment]
Yep the "with" options are a good idea to raise visibility.
With the distro macros I was a bit undecided as when disabled merely everywhere nobody would have a look probably ;-)

**Comment by wenningerk** [comment]
made syncing with pacemaker configurable via with/without for the spec-file.
defaulting now to enabled syncing as well if nothing passed to configure to go the whole way ;-)

File: configure.ac [hunk 1]
```diff
@@ -332,15 +332,15 @@ AC_DEFINE_UNQUOTED(SBD_WATCHDOG_TIMEOUT_DEFAULT, $SBD_WATCHDOG_TIMEOUT_DEFAULT,
...
+    SBD_SYNC_RESOURCE_STARTUP_DEFAULT=yes
 fi
 AC_SUBST(SBD_SYNC_RESOURCE_STARTUP_DEFAULT)
 dnl rather pass to C as a string and interpret there for consistent interpretation
 AC_DEFINE_UNQUOTED(SBD_SYNC_RESOURCE_STARTUP_DEFAULT, "${SBD_SYNC_RESOURCE_STARTUP_DEFAULT}",
     Default for SBD_SYNC_RESOURCE_STARTUP if not given in SBD configuration file explicitly)
 
 if test x"${SBD_SYNC_RESOURCE_STARTUP_SYSCONFIG}" = x""; then
-    SBD_SYNC_RESOURCE_STARTUP_SYSCONFIG=no
+    SBD_SYNC_RESOURCE_STARTUP_SYSCONFIG=yes
```
Comment (kgaillot) [inline configure.ac:343]: Not important, but SBD_SYNC_RESOURCE_STARTUP_SYSCONFIG=${SBD_SYNC_RESOURCE_STARTUP_DEFAULT} might make sense (i.e. default to the default)

**Comment by kgaillot** [comment]
Looks good to me

File: configure.ac (hunk 1 above)
Comment (wenningerk) [reply configure.ac:343]: good point 

definitely a logic that makes sense
but doesn't map very well to the with/without-logic in the spec-file

maybe having just the default exposed for with/without would make sense +
in addition a global for the sysconfig-value that is empty/auto/yes/no to
still allow explicit setting

**Comment by wenningerk** [comment]
little more description and more useful defaulting
on the pacemaker-syncing config

```
# Be careful with sync_resource_startup_default
# being enabled. This configuration has
# to be in sync with configuration in pacemaker
# where it is called sbd_sync - assure by e.g.
# mutual rpm dependencies.
%bcond_without sync_resource_startup_default
# Syncing enabled per default will lead to
# syncing enabled on upgrade without adaption
# of the config.
# Setting can still be overruled via sysconfig.
# The setting in the config-template packaged
# will follow the default if below is is left
# empty. But it is possible to have the setting
# in the config-template deviate from the default
# by setting below to an explicit 'yes' or 'no'.
%global sync_resource_startup_sysconfig ""
```

**Comment by kgaillot** [comment]
Looks good

**Pull request merged**